views/scan.hbs

```diff
@@ -8,9 +8,21 @@
       {{#if specificBreach}}
         {{#getScanResultsHeadline }}
           <h2 class="headline scan-results-headline fb-res-headline">{{{ this.headline }}}</h2>
-          {{#if this.subhead }}
-            <h3 class="scan-res-subhead">{{{ this.subhead }}}</h2>
-          {{/if}}
+            {{#ifCompare ../specificBreach.Domain "===" "facebook.com"}}
+            <div class="facebook-notes">
+            <p>Your email doesn’t appear in this breach, but your phone number may still be vulnerable. Some of the accounts compromised in the Facebook breach included phone numbers and other personal information but not email addresses. If you have ever signed up for a Facebook account -- even if you don’t use it now -- we recommend you take these three simple steps to protect yourself:
+</p>
+            <h3>What to do</h3>
+            <ol>
+            <li><strong>Change the passcode on your mobile phone carrier accounts to prevent SIM-jacking.</strong> SIM-jacking is when a hacker uses phone numbers, date of birth and other data to take over a person’s cell phone number and then hack into their email, social media and even financial accounts.</li>
+            <li><strong>Change your profile information to private in your Facebook privacy settings.</strong> During this breach, hackers took profile information that was set as “open to the public” or “shared with friends.” This information can be combined with other data to access even more of your personal information and accounts.</li>
+            <li>Follow the recommendations on <a href="/breach-details/Facebook">our Facebook breach page</a>.</li>
```

Comment:
Now that HIBP supports searching by phone numbers (I'm not clear if that works for ALL breaches or just Facebook), should we point people to https://haveibeenpwned.com/ to search for their phone number?

Reply:
Also, is it weird if the page is localized, but we're hardcoding a big block of English in the templates?

Reply:
Good points. Let's update and translate the CTAs to send people to https://blog.mozilla.org/firefox/facebook-data-leak-explained/ and https://blog.mozilla.org/firefox/mozilla-explains-sim-swapping/.

Reply:
Oh, and HIBP supports searching phone numbers for this breach only. :/

Reply:
Fixed.